src/pages/about.astro

```diff
@@ -3,29 +3,124 @@
 // SPDX-License-Identifier: Apache-2.0
 
 import Blob from "../components/Blob.astro";
+import Button from "../components/Button.astro";
 import Layout from "../layouts/Layout.astro";
+import MiniLeaderboard from "../components/MiniLeaderboard.astro";
+import TextButton from "../components/TextButton.astro";
+import {
+  getMembers, filterInactiveMembers,
+} from '../members.ts';
+
+const members = filterInactiveMembers(await getMembers());
 ---
 
 <Layout>
   <div class="edge-blobs">
     <Blob kind="solid-fill-03" top="72%" left="-13rem"></Blob>
+    <Blob kind="grad-dots-04" top="8%" right="-3rem"></Blob>
     <Blob kind="grad-dots-06" top="37%" right="-5rem"></Blob>
   </div>
   <div class="container">
     <Blob kind="solid-fill-05" top="20%" left="-16rem"></Blob>
     <Blob kind="solid-fill-04" top="90%" right="-20rem"></Blob>
+    <Blob kind="grad-dots-03" top="10%" left="-15rem"></Blob>
 
     <main id="main-content">
+      <h1>About the Pledge</h1>
+
+      <section>
+        <h2>What is the Open Source Pledge?</h2>
+
+        <p
+          >Open Source Pledge is a group of companies with a shared commitment to paying the maintainers of the Open
+          Source software we all consume. Our goal is to establish a new social norm in the tech industry of companies
+          paying Open Source maintainers, so that burnout and related security issues such as those in XZ and Log4j can
+          become a thing of the past.</p
+        >
+      </section>
+
+      <section>
+        <h2>What does making the Pledge entail?</h2>
+
+        <p
+          >Our new norm is for companies to pay a minimum of $2000 per year per full-time equivalent developer on the
+          company's staff, to Open Source maintainers of each company's choosing.</p
+        >
+      </section>
+
+      <section>
+        <h2
+          >Can companies become eligible by hiring developers to work on Open Source software, or giving gifts such as
+          cloud computing credits?</h2
```

Comment:
Kinda wordy. How about this?

```suggestion
          >What about other ways to support Open Source?</h2
```

Reply:
@chadwhitacre See: https://github.com/opensourcepledge/opensourcepledge.com/pull/119/commits/57c90660e94f7fedec5e2ca45d46097d56a5c09d